spec/requests/redirector_spec.rb

```diff
@@ -82,5 +82,15 @@
       expect(response.location).to be_in(['/target-two?page=2', '/target-one?lang=es'])
       expect(response.code).to eq('301')
     end
+
+    it 'truncates to the first five params for calculation but passes on all params' do
+      create(:location, :path => '/test?lang=es', :canonical => false, :resource => resource)
+
+      get '/test?lang=es&param2=2&param3=3&param4=4&param5=5&param6=6'
+      expect(response).to redirect_to('/es/target?param2=2&param3=3&param4=4&param5=5&param6=6')
+
+      get '/test?param2=2&param3=3&param4=4&param5=5&param6=6&lang=es'
+      expect(response).to redirect_to('/es/target?param2=2&param3=3&param4=4&param5=5&param6=6&lang=es')
```

Comment:
@benhutton Is this the best way to prove this is happening? It's a little confusing.. it took me a few minutes to work this out. I'd say we either need a comment to explain it or a different approach.